tests/unit/helpers/format-time-test.ts

```diff
@@ -47,35 +53,35 @@ module('format-time-test', function (hooks) {
 
     // Must provide `timeZone` because: https://github.com/ember-intl/ember-intl/issues/21
     await render(hbs`{{format-time dateString timeZone='UTC'}}`);
-    assert.equal(this.element.textContent, '1/23/2014');
+    assert.equal(this.element.textContent, '11:00 PM');
   });
 
   test('it should return a formatted string formatted with formatConfig key', async function (this: TestContext, assert) {
     assert.expect(1);
 
     this.intl.set('formats', {
-      time: { example: { units: 'hour', style: 'numeric', timeZone: 'utc', timeZoneName: 'short' } },
+      time: { example: { timeZone: 'utc', timeZoneName: 'short' } },
     });
 
     this.set('dateString', 'Thu Jan 23 2014 18:00:44 GMT-0500 (EST)');
 
     // Must provide `timeZone` because: https://github.com/ember-intl/ember-intl/issues/21
     await render(hbs`{{format-time dateString format='example'}}`);
-    assert.equal(this.element.textContent, '1/23/2014, UTC');
+    assert.equal(this.element.textContent, '11:00 PM UTC');
```

Comment:
Curious why the test output changed and if there was a need to remove the `units`/`style` from above time format.  Worth understanding since I may need to document if there's a breaking change here.

Reply:
it's bc the default is different in formatjs, which makes more sense to me. Previously as u can see `format-time` is triggered but the output is a date. The default for `formatTime` in formatjs will output numeric time instead so things like `today is {ts, time}` output actual time and `today is {ts, date}` output date. Seems like the way it works rn in ember-intl is that `format-time` & `format-date` do the same thing and they are meant to be semantically different in ICU (& in translation too).

Reply:
Thanks, I'll note this in the upgrade docs.